server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -624,6 +624,12 @@ public static IndexMergePolicy fromString(String text) {
         Property.IndexScope
     );
 
+    public static final Setting<Boolean> INDEX_CHECK_PENDING_FLUSH_ENABLED = Setting.boolSetting(
```

Comment:
Please add the javadoc (as per Apache Lucene 9.11.0 snapshot):

```
  /**
   * Expert: Returns if indexing threads check for pending flushes on update in order to help our
   * flushing indexing buffers to disk. This is an experimental Apache Lucene feature.
   */
```

Reply:
Thanks! Added